src/js/iqons.js

```diff
@@ -169,20 +169,22 @@ ${ content }
     }
 
     static _hash(text) {
-        const hash = ('' + text
+        const fullHash = ('' + text
                 .split('')
                 .map(c => Number(c.charCodeAt(0)) + 3)
                 .reduce((a, e) => a * (1 - a) * this.__chaosHash(e), 0.5))
             .split('')
-            .reduce((a, e) => e + a, '')
-            .substr(4, 17)
-            .replace('.', '0'); // A dot cannot be parsed into an int
+            .reduce((a, e) => e + a, '');
+
+        const hash = fullHash
+            .replace('.', fullHash[5]) // Replace the dot as it cannot be parsed to int
```

Comment:
Comment about why `fullHash[5]` is used would be good.

Also could `fullHash[5]` also be a `.` if we're unlucky?

Not using same `fullHash[5]` for the dot and all pad positions would ensure a greater variability.

Reply:
Yes, the number could even be `0.23582...` if we're unlucky.
I did some testing and it seems like larger strings generate larger numbers though.
So hashing the hash again might cause problems.

Reply:
The chance of `fullHash[5]` being `.` is probably something like 1 in 100 million